mypy/nodes.py

```diff
@@ -416,6 +416,7 @@ class FuncItem(FuncBase):
     # Is this an overload variant of function with more than one overload variant?
     is_overload = False
     is_generator = False   # Contains a yield statement?
+    is_coroutine = False   # Defined using 'async def'?
```

Comment:
That seems fine to me.  It might be worth making that even more explicit in this comment (so future readers won't wonder if the decorator case was just accidentally not mentioned).